modules/custom/gcnotify/gcnotify.module

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Contains gcnotify.module.
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+
+/**
+ * Implements hook_help().
+ */
+function gcnotify_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    // Main module help for the gcnotify module.
+    case 'help.page.gcnotify':
```

Comment:
@RabiaSajjad where does the `help.page.gcnotify` route get set?

Reply:
Ah okay, sounds good. Yeah I really like having any help/documentation pages right there in the admin interface, so it could help anyone figure out how to use the GC Notify Status view and filters. +1 for using the Help module from me!